### ISSUE TITLE
[BUG] Schema Editor - error on field size change

### ISSUE THREAD

**Issue by XenBG** [TiagoSilvaPereira/vemto-issues#39]
```
Error: Item with identifier 5 not found on table fields
    at e.exports.findOrFail (app://./js/chunk-vendors.40cad573.js:389:2577)
    at e.exports.update (app://./js/chunk-vendors.40cad573.js:389:2746)
    at Proxy.update (app://./js/chunk-vendors.40cad573.js:57:3254)
    at Proxy.save (app://./js/chunk-vendors.40cad573.js:57:2969)
    at a.save (app://./js/project_schema.34a72026.js:1:26187)
    at a.<anonymous> (app://./js/project_schema.34a72026.js:1:26067)
    at v (app://./js/project_crud~project_schema.ade7bb6f.js:9:49580)
    at y (app://./js/project_crud~project_schema.ade7bb6f.js:9:49844)
    at b (app://./js/project_crud~project_schema.ade7bb6f.js:9:49796)
```

I'm getting an error when I'm trying to change table field size from defaults (255 for a string) to something else.

Image: https://i.imgur.com/ecDQhYz.png

---

vemto 0.5.9
Electron 8.5.2
linux 5.4.0-54-generic
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Hi @XenBG thank you for reporting, I'll fix this issue for the next Vemto version. Cheers

**Comment by TiagoSilvaPereira** [comment]
Hi @XenBG this issue has fixed on **0.5.10**... Thanks
